### ISSUE TITLE
setting email seems to be buggy

### ISSUE THREAD

**Issue by bobheadxi** [ubclaunchpad/rocket2#488]
**Describe the bug**

**To Reproduce**

1. `/rocket user edit --email robert@bobheadxi.dev`
2. Email registers as `obheadxi.dev`

**Expected behavior**

should register email. This seems to work if I quote:

```
/rocket user edit --email 'robert@bobheadxi.dev'
```

**Screenshots**

![image](https://user-images.githubusercontent.com/23356519/94217825-8640a100-ff15-11ea-9a80-eabaf8b6a034.png)

**Additional context**

**Comment by cheukyin699** [comment]
That's an... interesting bug.

**Comment by bobheadxi** [comment]
Actually, I'm not sure if quoting fixes:

```
{Time: 2020-09-25 10:08:34, Level: [INFO], function: server.handle_commands():113, message: @U01C0BGLVL0: /rocket2 user edit --email 'robert@bobheadxi.dev'}
{Time: 2020-09-25 10:08:34, Level: [INFO], function: dynamodb.store():203, message: Storing obj {'slack_id': 'U01C0BGLVL0', 'name': '', 'email': 'obheadxi.dev', 'github_username': '', 'github_id': '', 'major': '', 'position': '', 'biography': '', 'image_url': '', 'permissions_level': <Permissions.admin: 3>, 'karma': 1} in table users}
```

**Comment by cheukyin699** [comment]
Issue is in the email escaping function, that escapes emails irregardless whether it needs escaping. Will be fixed in #498.
